src/utils.js

```diff
@@ -180,6 +223,8 @@ const FOCUSABLE_SELECTOR = [
   'button:not([disabled])',
   'select:not([disabled])',
   'textarea:not([disabled])',
+  'div[contenteditable=""]',
+  'div[contenteditable="true"]',
```

Comment:
Does it have to be a `div`? Couldn't it be a `span` or anything else?

```suggestion
  '[contenteditable=""]',
  '[contenteditable="true"]',
```

Reply:
😓